packages/core/admin/admin/src/content-manager/pages/ListView/components/FieldPicker/index.js

```diff
@@ -1,81 +1,99 @@
 import React from 'react';
 
-import { Select, Option, Box } from '@strapi/design-system';
-import { useTracking } from '@strapi/helper-plugin';
+import { Box, Flex, Checkbox, TextButton, Typography } from '@strapi/design-system';
+import { useCollator, useTracking } from '@strapi/helper-plugin';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
 
-import { getTrad, checkIfAttributeIsDisplayable } from '../../../../utils';
-import { onChangeListHeaders } from '../../actions';
+import { checkIfAttributeIsDisplayable } from '../../../../utils';
+import { onChangeListHeaders, onResetListHeaders } from '../../actions';
 import { selectDisplayedHeaders } from '../../selectors';
 
+const StyledCheckboxLabel = styled(Box)`
+  label {
+    font-size: 0.75rem;
+    div {
+      flex-wrap: wrap;
+      display: flex;
+    }
+  }
```

Comment:
Could you please elaborate what problem this is solving? :)

Reply:
Yes sure
This is the version before my fixes
<img width="254" alt="before" src="https://github.com/strapi/strapi/assets/2589748/7d80ef38-bdd3-4632-82c2-63e53ae19789">
and this one afterwards
<img width="252" alt="after" src="https://github.com/strapi/strapi/assets/2589748/60c5a075-3da1-4e2f-bcbb-042f516d463c">

We changed the font size to have the same size as the "Configure the view" button, the Diplayed fields title and the Reset button.
Also as you can see in the first image, the content inside the checkbox wrapper wasn't aligned vertically.

Reply:
Ah, thanks.

> font-size: 0.75rem;

For this: could you please use a theme value over a hardcoded font-size?

> div {
>      flex-wrap: wrap;
>      display: flex;
> }

This this I am afraid we should find another solution. I think you could go two ways:

1) Improve the DS component to make sure the label is aligned regardless of the font-size
2) (preferred) use `BaseCheckbox` over `Checkbox` and use the `Typography` component to set the proper font-size and a `Flex` wrapper for the alignment.

I believe solution 2) would also address the font-size topic from above.

Does that make sense?

Reply:
yes sure I can try, thanks @gu-stav